middlewares/validator.js

```diff
@@ -0,0 +1,9 @@
+const validateData = (validator) => {
+  return (req, res, next) => {
+    const { error } = validator(req.body);
+    if (error) return res.status(400).json(error.details[0].message);
```

Comment:
dla zabawy i wiedzy :-] mozesz tez tak:
```js
if ({ details: [{ message }] }) { 
  return res.status(400).json(message);
}
```
funkcyjne jezyki lubia tak podchodzic do zabawy z danymi. W JS to mniej popularne, ale (Dla mnie) wiecej, niz dobre 😁